modules/installation-ibm-z-troubleshooting-and-debugging.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * installing/installing_ibm_z/installing-ibm-z.adoc
+
+[id="installation-ibm-z-troubleshooting-and-debugging_{context}"]
+= Completing installation on user-provisioned infrastructure
+
+You can gather the debugging information that might help you to troubleshoot and
```

Comment:
the debugging information -> debugging information

Reply:
Fixed.